modules/machine-user-infra-machines-ibm-z.adoc

```diff
@@ -102,8 +102,8 @@ The following is an example parameter file, `additional-worker-dasd.parm`:
 rd.neednet=1 \
 console=ttysclp0 \
 coreos.inst.install_dev=/dev/dasda \
-coreos.live.rootfs_url=http://cl1.provide.example.com:8080/assets/rhcos-live-rootfs.s390x.img \
-coreos.inst.ignition_url=http://cl1.provide.example.com:8080/ignition/worker.ign \
+coreos.live.rootfs_url=http://<http_server>/rhcos-<version>-live-rootfs.<architecture>.img \
+coreos.inst.ignition_url=http://<http_server>/worker.ign \
 ip=172.18.78.2::172.18.78.1:255.255.255.0:::none nameserver=172.18.78.1 \
```

Comment:
We are using hardcoded vales for IP, netmask,ect .. why we are not using the same variables as in the KVM section above
```suggestion
ip=<ip>::<default_gateway>:<subnet_mask_length>:<hostname>::none:<MTU> nameserver<dns> \
 ```